docs/guidance/identifiers.md

```diff
@@ -33,7 +33,7 @@ There are different approaches to including project identifiers in contracting d
 
 In OCDS, the identifier for the individual infrastructure project to which a contracting process is related should be disclosed using the `planning/project/id` field, introduced in the [Budget and Projects extension](https://extensions.open-contracting.org/en/extensions/budget_project/).
 
-The `planning/budget/projectID` field in OCDS should **not** be used to disclose the identifier for an individual infrastructure project. This field is used to disclose the identifier for a project in the national budget, which might include multiple infrastructure projects to which the contracting process is related.
+The `planning/budget/projectID` field in OCDS should **not** be used to disclose the identifier for an individual infrastructure project. This field is used to disclose the identifier for a project in the national budget, which might include multiple infrastructure projects, to which the contracting process is related.
```

Comment:
I think I am still confused. Can we instead make two sentences from the last sentence?

Reply:
I've split them out in https://github.com/open-contracting/infrastructure/pull/191/commits/fb43a7931ac5bfe52dc75ecd7b43ce110bb25c2e and explained why it is necessary to disclose both identifiers.